.circleci/config.yml

```diff
@@ -20,11 +20,11 @@ jobs:
           command: yarn run build-all
       - run:
           name: Run the site server
-          command: PORT=80 ./bin/serve
+          command: PORT=8016 ./bin/serve
           background: true
       - run:
           name: Check site is accessible
-          command: sleep 4 && curl --head --fail --retry-delay 5 --retry 10  --retry-connrefused http://localhost
+          command: sleep 4 && curl --head --fail --retry-delay 5 --retry 10  --retry-connrefused http://localhost:8016
```

Comment:
Why did you do this? I normally test all these sites on `80` to be more production-like.